doc/api/process.markdown

```diff
@@ -59,6 +59,9 @@ finished running the process will exit. Therefore you **must** only perform
 checks on the module's state (like for unit tests). The callback takes one
 argument, the code the process is exiting with.
 
+This event is not guaranteed to fire in all situations. For example, if the
+process receives `SIGINT`, it may exit without firing this event.
```

Comment:
well no nevermind, it's a much different signal from `ctrl` + `c` stuff.

`SIGHUP` is any hangup, so while it could happen in the terminal, i think it could also occur from any socket opened by `http` or `net` or a binding, etc